images/tumbleweed/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM opensuse/tumbleweed:latest
```

Comment:
The Dockerfile has to be in the `images/opensuse` directory: it's setup to take the image name from the repository name. At the moment it's building `quay.io/footloose/tumbleweed` and so the tests fail to pick up the image.

Reply:
What should I change?

Reply:
Let's try this.. (pushing again)